lms/djangoapps/teams/tests/test_views.py

```diff
@@ -777,6 +785,40 @@ def test_access_by_username(self, user, status, has_content):
             else:
                 self.assertEqual(membership['count'], 0)
 
+    @ddt.data(
+        ('student_enrolled_both_courses_other_team', 'TestX/TS101/Test_Course', 200, 'Nuclear Team'),
+        ('student_enrolled_both_courses_other_team', 'MIT/6.002x/Circuits', 200, 'Another Team'),
+        ('student_enrolled', 'TestX/TS101/Test_Course', 200, u'sólar team'),
+        ('student_enrolled', 'MIT/6.002x/Circuits', 400, ''),
+    )
+    @ddt.unpack
+    def test_course_filter_with_username(self, user, course_id, status, team_name):
```

Comment:
Nitpick: I'd like to see `status` come last for readability, as I tend to read these from right to left: Something along the lines of 'when we test with this user, this test course, this team name, we should expect this status'. (nitpick means you don't have to address this comment :) )